ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -85,6 +85,13 @@ if tf_support_is_available
 endif
 
 if tflite_support_is_available
+  if not get_option('flatbuf-support').disabled()
+    flatbuf_dep = cc.find_library('flatbuffers', required: false)
```

Comment:
Is it necessary?
flatbuf_dep is already defined in https://github.com/nnstreamer/nnstreamer/blob/3362d11a458bfd38d5b4e55f5a79694438d741e5/meson.build#L147

Reply:
Nice catch. I will remove this line.